catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RegistryPublicationServiceImpl.java

```diff
@@ -151,6 +163,10 @@ public void update(Metacard metacard) throws FederationAdminException {
                 //also and the end result will be the same.
                 federationAdminService.addRegistryEntry(metacard, locations);
             }
+            LOGGER.info("Updated publication for registry entry {}:{} at {}",
```

Comment:
I like the futuristic logging - I'm about to update registry entry {}:{} at {} - then if it fails you know what you were trying to do. Move each of these up before the action?